packages/i18n/src/locales/en.i18n.json

```diff
@@ -2304,6 +2304,9 @@
   "Federation_Matrix_max_size_of_public_rooms_users": "Maximum number of users when joining a public room in a remote server",
   "Federation_Matrix_max_size_of_public_rooms_users_desc": "The number of the maximum users when joining a public room in a remote server. Public Rooms with more users will be ignored in the list of Public Rooms to join.",
   "Federation_Matrix_max_size_of_public_rooms_users_Alert": "Keep in mind, that the bigger the room you allow for users to join, the more time it will take to join that room, besides the amount of resource it will use. <a target=\"_blank\" href=\"https://matrix.org/blog/2022/10/18/testing-faster-remote-room-joins\">Read more</a>",
+  "Federation_Matrix_serve_well_known": "Serve Well Known",
+  "Federation_Matrix_serve_well_known_desc": "Serve /.well-known/matrix/server and /.well-known/matrix/client directly from within Rocket.Chat instead of reverse proxy for federation",
```

Comment:
```suggestion
  "Federation_Matrix_serve_well_known_Description": "Serve /.well-known/matrix/server and /.well-known/matrix/client directly from within Rocket.Chat instead of reverse proxy for federation",
```